content/20.treatments.md

```diff
@@ -1,260 +1,195 @@
-In addition to these independent trials, Roche, in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial in early April.
-The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyze its efficiency/safety [@url:https://www.biopharma-reporter.com/Article/2020/03/19/Roche-enters-Phase-III-for-COVID-19-treatment].
-However, previous studies of RA showed that the rate of incident infections in clinical practice patients treated with TCZ was higher than the rate observed during clinical trial [@doi:10.1093/rheumatology/ker223].
-Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs [@doi:10.1111/1756-185X.13010].
-These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
-This aspect is probably crucial to be investigated further in upcoming trials.
+### Small Molecule Drugs
 
-Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the COVID-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.
-<!-##### Summary
+#### Small Molecule Drugs for Targeting SARS-CoV-2
 
-Summarize the state of the symptom management approach.
--->
-
-### Small Molecule Drugs for Targeting SARS-CoV-2
-
-The replication cycle of a virus within an epithelial host cell includes six basic steps that can be summarized as follows: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457].
+The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457].
```

Comment:
```suggestion
The replication cycle of a virus within an epithelial host cell includes 6 basic steps that can be briefly summarized as follows: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457].
```
Just a heads up, this will likely get moved to the intro within the next week or two, we are working on restructuring this type of content